src/libraries/Microsoft.PowerFx.Core/Binding/Binder.cs

```diff
@@ -1681,7 +1681,23 @@ public IEnumerable<FirstNameInfo> GetFirstNamesInTree(TexlNode node)
 
         public IEnumerable<FirstNameInfo> GetFirstNames()
         {
-            return _infoMap.OfType<FirstNameInfo>();
+            var id = 0;
```

Comment:
This seems like the wrong approach. Rather, Intellisense should be able to pull these names either from a resolver it has access to, or PowerFxConfig or something like that.

Get*Names() methods on binder are for retrieving those within the bound AST, not for also passing through the symbol table.